### ISSUE TITLE
Eclipse download site is not accessible

### ISSUE THREAD

**Issue by jschewebbn** [Protelis/Protelis#228]
According to https://github.com/Protelis/Protelis I should be able to use `http://efesto.apice.unibo.it/protelis-build/protelis-parser/protelis.parser.repository/target/repository/` as the repository. When I do this I receive an error that the repository is not reachable.

```
Unable to read repository at http://efesto.apice.unibo.it/protelis-build/protelis-parser/protelis.parser.repository/target/repository/content.xml.
Unable to read repository at http://efesto.apice.unibo.it/protelis-build/protelis-parser/protelis.parser.repository/target/repository/content.xml.
No route to host (Host unreachable)
```

I'm trying to make sure that I have the latest plugin version to validate errors that I have been seeing.

**Comment by DanySK** [comment]
Hi Jon,
we need to update the website. Protelis artifacts are now hosted on [the official Eclipse Marketplace](https://marketplace.eclipse.org/content/protelis).

Please don't close, let's keep this open until the documentation is updated.

**Comment by jakebeal** [comment]
Note that the install warns about "unsigned content" - is that something that can be fixed, given that it's in the official marketplace?

**Comment by DanySK** [comment]
I never investigated. I don't think artifact signing is related to the host (you can sign an artifact and publish it on your own site, or not sign and publish on the Eclipse marketplace).

**Comment by jakebeal** [comment]
It would be nice to have signatures, so that people know it's not random malware.

**Comment by jschewebbn** [comment]
I had found the site, but couldn't remember how to get it installed. I keep forgetting that Eclipse uses the marketplace now even though the old install from repository option is still there.

**Comment by DanySK** [comment]
Well @jakebeal, it comes from the Eclipse Marketplace, there is some degree of  trust that it's not malware. I can investigate signatures though, if you believe it's important.

**Comment by jakebeal** [comment]
I wouldn't put signatures as top priority, but I do think it would be nice to be able to sign our software.

**Comment by jakebeal** [comment]
Signatures is extracted as a subissue, and this is now a documentation issue.

**Comment by DanySK** [comment]
We need to update the readme of protelis and clearly state that Eclipse artifacts are now on the marketplace

**Comment by DanySK** [comment]
Fixed by #266

**Comment by DanySK** [comment]
Fixed in [13.1.0](https://github.com/Protelis/Protelis/releases/tag/13.1.0)
